### PR TITLE
no changes to previous master

### PR DESCRIPTION
**Pull request by alexandraschlegel**

### CODE REVIEW

**Comment by Haumer** [comment]
this file looks all good! 
is there something I am missing?

**Comment by alexandraschlegel** [comment]
So the problem is the still the same as this morning, so there is something wrong with the order-item controller. If you could have a look!

**Comment by alexandraschlegel** [comment]
the add to basket button on the product show page doesn't lead to the next page

**Comment by alexandraschlegel** [comment]
irgendwie stimmt die Logik ja nicht, es muesste doch eigentlich von der Product page zur cart-item page gehen und nicht zur Order-Item Page

File: app/views/products/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <h1>Products</h1>
+  <h1>Products list</h1>
```
Comment (Haumer) [inline app/views/products/index.html.erb:2]: 👍

**Pull request merged**